<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-TRU5U6IG" failures="1" tests="3" name="basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4" time="80.088" errors="0" timestamp="2024-06-26T20:02:26 IST" skipped="0">
  <testcase classname="basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4" name="test4" time="34.475">
    <failure message="tittle not matching expected [Amazon.in: Mobile Cover - Include Out Of Stock] but found [Amazon.in : mobile cover]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: tittle not matching expected [Amazon.in: Mobile Cover - Include Out Of Stock] but found [Amazon.in : mobile cover]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4.test4(searchMobileCoverTc4.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
]]>
    </failure>
  </testcase> <!-- test4 -->
  <system-out/>
  <testcase classname="basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4" name="test4" time="23.405"/>
  <system-out/>
  <testcase classname="basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4" name="test4" time="22.208"/>
  <system-out/>
</testsuite> <!-- basic.MavenProject.amazonScenario10Tc.searchMobileCoverTc4 -->
